queries/perl/highlights.scm

```diff
@@ -1,198 +1,89 @@
-; Misc keywords
 [
```

Comment:
Keep the previous captures for these:

```scheme
[ "use" "no" "require" ] @include

[ "if" "elsif" "unless" "else" ] @conditional

[ "while" "until" "for" "foreach" ] @repeat

"return" @keyword.return

"sub" @keyword.function
```

Also, "map" and "grep" could be `@function.builtin`.